### ISSUE TITLE
[Bug]: MangoHud doesn't work in the Flatpak version of Bottles v51.14

### ISSUE THREAD

**Issue by RedSnt** [bottlesdevs/Bottles#3529]
### Describe the bug

After updating from 51.13 to 51.14, MangoHud no longer works ingame for me. It used to work, and it still works for other flatpak games (like prism launcher -> minecraft).

### To Reproduce

Make sure `org.freedesktop.Platform.VulkanLayer.MangoHud` is installed in flatpak, then make sure `MANGOHUD=1` environment variable is set in flatseal for Bottles or all applications as in my case. Less important step is setting up so that Mangohud.conf is visible for flatpaks by exposing `xdf-config/MangoHud:ro` under filesystem/other files.  

Now run Bottles and try running anything 3D like a game where MangoHud would normally initialize. In my case both Civilization 4 and Fallout New Vegas fail, no mangohud running.

### Package

Flatpak from Flathub

### Distribution

MX Linux 23.4 (Debian Bookworm-based)

### Debugging Information

```shell
Official Package: true
Version: '51.14'
DE/WM: plasma
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        nvidia:
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
            nvngx_path: /usr/lib/x86_64-linux-gnu/GL/nvidia-560-35-03/extra/nvidia/wine
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.11.3-1-liquorix-amd64
Disk:
    Total: 33630425088
    Free: 33630244864
RAM:
    MemTotal: 62.6GiB
    MemAvailable: 53.9GiB
Bottles_envs: null
```

### Troubleshooting Logs

```shell
Not sure it'll reveal anything, but here's the log starting Bottles, running Civ4, then closing it and then closing Bottles:

redsnt@roedehelgen:~
$ flatpak run com.usebottles.bottles

(process:2): Gtk-WARNING **: 15:44:35.773: Locale not supported by C library.
        Using the fallback 'C' locale.

(process:2): Gtk-WARNING **: 15:44:36.083: Unknown key gtk-modules in /home/redsnt/.var/app/com.usebottles.bottles/config/gtk-4.0/settings.ini

(process:2): Adwaita-WARNING **: 15:44:36.095: Using GtkSettings:gtk-application-prefer-dark-theme with libadwaita is unsupported. Please use AdwStyleManager:color-scheme instead.
15:44:36 (INFO) Launching with URI: None 
15:44:36 (INFO) [Activate] request received. 
15:44:36 (INFO) Bottles Started! 
15:44:37 (INFO) Performing Bottles checks… 
15:44:37 (INFO) Dxvks found:
         - dxvk-2.4
 
15:44:37 (INFO) Vkd3ds found:
         - vkd3d-proton-2.13
 
15:44:37 (INFO) Nvapis found:
         - dxvk-nvapi-v0.7.1
 
15:44:37 (INFO) Latencyflexs found:
         - latencyflex-v0.1.1
 
15:44:37 (INFO) Runners found:
         - soda-9.0-1
         - wine-ge-proton8-26
         - kron4ek-wine-proton-9.0-3-amd64
         - kron4ek-wine-proton-9.0-2-amd64
         - sys-wine-9.0
 
15:44:37 (ERROR) Config file not exists 
Traceback (most recent call last):
  File "/app/share/bottles/bottles/backend/models/config.py", line 183, in load
    raise FileNotFoundError("Config file not exists")
FileNotFoundError: Config file not exists
15:44:37 (INFO) Bottles found:
         - cp2077
         - wacup
         - Civ 4
         - Fonv
         - rando
 
15:44:37 (INFO) Catalog dependencies loaded 
15:44:37 (INFO) Catalog installers loaded 
15:44:37 (INFO) Catalog components loaded 
15:44:42 (INFO) Setting Key sync=fsync for bottle Civ 4… 
15:44:42 (INFO) There is no running wineserver. 
15:44:44 (INFO) Launching an executable… 
15:44:44 (INFO) Using EasyAntiCheat runtime 
15:44:44 (INFO) Using BattlEye runtime 
fsync: up and running.
wine: setpriority -10 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
info:  Game: Civilization4.exe
info:  DXVK: v2.4
info:  Found config file: /home/redsnt/.config/dxvk.conf
info:  Effective configuration:
info:    dxvk.tearFree = Auto
info:    d3d9.presentInterval = 0
info:    dxgi.syncInterval = 0
info:    d3d9.numBackBuffers = 3
info:    dxgi.numBackBuffers = 3
info:    dxgi.hideNvkGpu = Auto
info:    dxgi.hideNvidiaGpu = Auto
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x79103950
info:  Built-in extension providers:
info:    Platform WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_EXT_surface_maintenance1
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Game: Civilization4.exe
info:  DXVK: v2.4
info:  Found config file: /home/redsnt/.config/dxvk.conf
info:  Effective configuration:
info:    dxvk.tearFree = Auto
info:    d3d9.presentInterval = 0
info:    dxgi.syncInterval = 0
info:    d3d9.numBackBuffers = 3
info:    dxgi.numBackBuffers = 3
info:    dxgi.hideNvkGpu = Auto
info:    dxgi.hideNvidiaGpu = Auto
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x79103950
info:  Built-in extension providers:
info:    Platform WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_EXT_surface_maintenance1
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Game: Civilization4.exe
info:  DXVK: v2.4
info:  Found config file: /home/redsnt/.config/dxvk.conf
info:  Effective configuration:
info:    dxvk.tearFree = Auto
info:    d3d9.presentInterval = 0
info:    dxgi.syncInterval = 0
info:    d3d9.numBackBuffers = 3
info:    dxgi.numBackBuffers = 3
info:    dxgi.hideNvkGpu = Auto
info:    dxgi.hideNvidiaGpu = Auto
info:  Vulkan: Found vkGetInstanceProcAddr in winevulkan.dll @ 0x79103950
info:  Built-in extension providers:
info:    Platform WSI
info:    OpenVR
info:    OpenXR
info:  OpenVR: could not open registry key, status 2
info:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_EXT_surface_maintenance1
info:    VK_KHR_get_surface_capabilities2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  Process set as DPI aware
info:  Device properties:
info:    Device : NVIDIA GeForce RTX 3070 Ti
info:    Driver : NVIDIA 560.35.3
info:  Enabled device extensions:
info:    VK_EXT_attachment_feedback_loop_layout
info:    VK_EXT_conservative_rasterization
info:    VK_EXT_custom_border_color
info:    VK_EXT_depth_bias_control
info:    VK_EXT_depth_clip_enable
info:    VK_EXT_extended_dynamic_state3
info:    VK_EXT_fragment_shader_interlock
info:    VK_EXT_graphics_pipeline_library
info:    VK_EXT_memory_priority
info:    VK_EXT_non_seamless_cube_map
info:    VK_EXT_robustness2
info:    VK_EXT_shader_module_identifier
info:    VK_EXT_swapchain_maintenance1
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_external_memory_win32
info:    VK_KHR_external_semaphore_win32
info:    VK_KHR_maintenance5
info:    VK_KHR_pipeline_library
info:    VK_KHR_present_id
info:    VK_KHR_present_wait
info:    VK_KHR_swapchain
info:    VK_KHR_win32_keyed_mutex
info:    VK_NV_descriptor_pool_overallocation
info:    VK_NV_raw_access_chains
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 0
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 0
info:    logicOp                                : 0
info:    multiDrawIndirect                      : 0
info:    drawIndirectFirstInstance              : 0
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    wideLines                              : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 0
info:    shaderImageGatherExtended              : 0
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 0
info:    shaderInt64                            : 0
info:    variableMultisampleRate                : 1
info:    shaderResourceResidency                : 0
info:    shaderResourceMinLod                   : 0
info:    sparseBinding                          : 0
info:    sparseResidencyBuffer                  : 0
info:    sparseResidencyImage2D                 : 0
info:    sparseResidencyImage3D                 : 0
info:    sparseResidency2Samples                : 0
info:    sparseResidency4Samples                : 0
info:    sparseResidency8Samples                : 0
info:    sparseResidency16Samples               : 0
info:    sparseResidencyAliased                 : 0
info:  Vulkan 1.1
info:    shaderDrawParameters                   : 0
info:  Vulkan 1.2
info:    samplerMirrorClampToEdge               : 1
info:    drawIndirectCount                      : 1
info:    samplerFilterMinmax                    : 0
info:    hostQueryReset                         : 1
info:    timelineSemaphore                      : 1
info:    bufferDeviceAddress                    : 0
info:    shaderOutputViewportIndex              : 1
info:    shaderOutputLayer                      : 1
info:    vulkanMemoryModel                      : 1
info:  Vulkan 1.3
info:    robustImageAccess                      : 0
info:    pipelineCreationCacheControl           : 1
info:    shaderDemoteToHelperInvocation         : 1
info:    shaderZeroInitializeWorkgroupMemory    : 0
info:    synchronization2                       : 1
info:    dynamicRendering                       : 1
info:  VK_AMD_shader_fragment_mask
info:    extension supported                    : 0
info:  VK_EXT_attachment_feedback_loop_layout
info:    attachmentFeedbackLoopLayout           : 1
info:  VK_EXT_conservative_rasterization
info:    extension supported                    : 1
info:  VK_EXT_custom_border_color
info:    customBorderColors                     : 1
info:    customBorderColorWithoutFormat         : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 1
info:  VK_EXT_depth_bias_control
info:    depthBiasControl                       : 1
info:    leastRepresentableValueForceUnormRepresentation : 1
info:    floatRepresentation                    : 0
info:    depthBiasExact                         : 1
info:  VK_EXT_extended_dynamic_state3
info:    extDynamicState3AlphaToCoverageEnable  : 1
info:    extDynamicState3DepthClipEnable        : 1
info:    extDynamicState3RasterizationSamples   : 1
info:    extDynamicState3SampleMask             : 1
info:    extDynamicState3LineRasterizationMode  : 1
info:  VK_EXT_fragment_shader_interlock
info:    fragmentShaderSampleInterlock          : 0
info:    fragmentShaderPixelInterlock           : 0
info:  VK_EXT_full_screen_exclusive
info:    extension supported                    : 0
info:  VK_EXT_graphics_pipeline_library
info:    graphicsPipelineLibrary                : 1
info:  VK_EXT_line_rasterization
info:    rectangularLines                       : 1
info:    smoothLines                            : 1
info:  VK_EXT_memory_budget
info:    extension supported                    : 1
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 1
info:  VK_EXT_non_seamless_cube_map
info:    nonSeamlessCubeMap                     : 1
info:  VK_EXT_robustness2
info:    robustBufferAccess2                    : 1
info:    robustImageAccess2                     : 1
info:    nullDescriptor                         : 1
info:  VK_EXT_shader_module_identifier
info:    shaderModuleIdentifier                 : 1
info:  VK_EXT_shader_stencil_export
info:    extension supported                    : 0
info:  VK_EXT_swapchain_colorspace
info:    extension supported                    : 0
info:  VK_EXT_swapchain_maintenance1
info:    swapchainMaintenance1                  : 1
info:  VK_EXT_hdr_metadata
info:    extension supported                    : 0
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 0
info:    geometryStreams                        : 0
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  VK_KHR_external_memory_win32
info:    extension supported                    : 1
info:  VK_KHR_external_semaphore_win32
info:    extension supported                    : 1
info:  VK_KHR_maintenance5
info:    maintenance5                           : 1
info:  VK_KHR_present_id
info:    presentId                              : 1
info:  VK_KHR_present_wait
info:    presentWait                            : 1
info:  VK_NV_descriptor_pool_overallocation
info:    descriptorPoolOverallocation           : 1
info:  VK_NV_raw_access_chains
info:    shaderRawAccessChains                  : 1
info:  VK_NVX_binary_import
info:    extension supported                    : 0
info:  VK_NVX_image_view_handle
info:    extension supported                    : 0
info:  VK_KHR_win32_keyed_mutex
info:    extension supported                    : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
info:    Sparse   : 0
info:  DXVK: Read 8 valid state cache entries
info:  DXVK: Graphics pipeline libraries supported
info:  D3D9DeviceEx::ResetSwapChain:
info:    Requested Presentation Parameters
info:      - Width:              1024
info:      - Height:             768
info:      - Format:             D3D9Format::X8R8G8B8
info:      - Auto Depth Stencil: true
info:                  ^ Format: D3D9Format::D24S8
info:      - Windowed:           false
info:      - Swap effect:        1
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Color space:  VK_COLOR_SPACE_SRGB_NONLINEAR_KHR
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR (dynamic: yes)
info:    Buffer size:  1030x800
info:    Image count:  3
info:    Exclusive FS: 0
info:  Setting display mode: 1024x768@0
info:  Setting display mode: 1024x768@0
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
00e0:err:kerberos:kerberos_LsaApInitializePackage no Kerberos support, expect problems
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
wine: setpriority 3 for pid -1 failed: 3
info:  DXVK: Using 16 compiler threads
err:   D3D9On12::GetD3D12Device: Stub
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state 16
warn:  D3D9SwapChainEx::QueryInterface: Unknown interface query
warn:  05eec05d-8f7d-4362-b999-d1baf357c704
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Color space:  VK_COLOR_SPACE_SRGB_NONLINEAR_KHR
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR (dynamic: yes)
info:    Buffer size:  1024x768
info:    Image count:  3
info:    Exclusive FS: 0
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state 161
wine: setpriority 2 for pid -1 failed: 3
wine: setpriority 2 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
wine: setpriority -10 for pid -1 failed: 3
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  D3D9: VK_FORMAT_D16_UNORM_S8_UINT -> VK_FORMAT_D24_UNORM_S8_UINT
info:  NVIDIA GeForce RTX 3070 Ti:
info:    Driver : NVIDIA 560.35.3
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[1]: Property Flags = 0x1
info:      Memory Type[2]: Property Flags = 0x1
info:      Memory Type[5]: Property Flags = 0x7
info:    Memory Heap[1]: 
info:      Size: 48108 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x6
info:      Memory Type[4]: Property Flags = 0xe
info:  Process set as DPI aware
00e0:err:x11drv:xinerama_get_fullscreen_monitors Failed to get xinerama fullscreen monitor indices.
00e0:err:x11drv:update_net_wm_fullscreen_monitors Failed to find xinerama monitors at (0,0)-(1024,768)
```

### Additional context

I'm reporting the bug here instead at flightlessmango/MangoHud because MangoHud still works in Flatpak as it used to, but stopped working for just Bottles, and it broke from the recent upgrade to 51.14, and I'm not sure what changed, because flatseal still says the same environments are set and same files exposed.  
It certainly would be nice if someone else can verify this or if I've made a mistake somehow.

**Comment by RedSnt** [comment]
Addendum, something I didn't think of before now: I tried downgrading to 51.13 and that made MangoHud start up correctly again, so seemingly it is a bug (or feature I'm overlooking?) introduced in 51.14. It's also present in 51.15.

**Comment by SjoerdV** [comment]
Had the same issue. fix is to install Mangohud 24.08 specifically.
```
flatpak install mangohud
# now choose the 24.08 version from the list, for me it was the top option
```

**Comment by SjoerdV** [comment]
> > Had the same issue. fix is to install Mangohud 24.08 specifically.
> > ```
> > flatpak install mangohud
> > # now choose the 24.08 version from the list, for me it was the top option
> > ```
> 
> Thanks, glad to hear I'm not imagining or have misconfigured something. I think I'll stick with a downgraded Bottles version for now, it's not like 51.14 and 51.15 changed a lot for my use case, and it's only really Fallout New Vegas that needs MangoHud's frame limiter feature.

Just found out the version dependencies also span to gamescope and vkBasalt (and vkBasalt does not have a 24.08 version available in flathub). So downgrading Bottles is the only option if you depend on vkBasalt. 
NOTE: you can actually view dependencies when hovering your mouse over the greyed out switch option in a Bottles bottle.

**Comment by mirkobrombin** [comment]
We updated the Runtime to a new version so, per as Flatpak works, you have to install all those components with the same version as pointed by SjoerdV.
